Topics/Tech_Stacks/Tanstack_Table.md

```diff
@@ -0,0 +1,141 @@
+# How to Build Customizable Tables in React
+
+## Introduction
+Tanstack Table is a modern, headless UI library for building powerful and interactive tables in React applications. Known for its flexibility and performance, it allows developers to create tables that can handle large datasets with ease. This article assumes you are working with React 17 or later, possibly in a project initialized with Create React App.
+
+## Why use it?
+This library is designed to be headless, meaning that it doesn't ship with any inherent styling, and instead relies on the developer to style all features related to the table. The benefit of this is that if your preferred UI library in React does not provide any built-in advanced table functionality, you can customize Tanstack table to fit all your needs easily without needing to override any preexisting styles.
```

Comment:
This is quite good, succinctly explains why tanstack is preferable over something like bootstrap or tailwind

Reply:
Thank you, glad to hear that this did help